### ISSUE TITLE
Add support for custom converters

### ISSUE THREAD

**Issue by CircuitSacul** [XuaTheGrate/Flag-Parsing#17]
I noticed that discord-flags doesn't support custom converters that are classes, for example:
```python
class Reversed(commands.Converter):
    async def convert(self, ctx, arg):
        return arg[::-1]
```

Using normal commands, I can do:
```python
@commands.command()
async def reverse(ctx, string: Reversed):
    await ctx.send(string)
```

But with discord-flags, it raises `argument --text: invalid convert value: 'test'`

Would it be possible to add support for custom converters?

**Comment by XuaTheGrate** [comment]
Fixed with #15

**Comment by CircuitSacul** [comment]
Thanks, but it looks like it was never updated on PyPI? I use this is a library and not as a submodule
